workflows/alevin-quant/run-alevin-fry.nf

```diff
@@ -150,12 +156,25 @@ workflow{
 
   barcodes_ch = samples_ch
     .map{row -> file("${params.barcode_dir}/${barcodes[row.technology]}")}
+  
+  // file paths
+  index_path = "${params.ref_dir}/${params.index_dir}/${index_names_map[params.index_type]}"
+  index_prefix = "${params.ref_dir}/${params.annotation_dir}"
+  t2g_path = "${index_prefix}/${t2g_map[params.index_type]}"
+  
+  // if using splici and cr-like use the 3 column t2g file for alevin-fry quant
+  if(params.resolution == 'cr-like' && params.index_type == 'splici'){
+      t2g_quant_path = "${index_prefix}/${params.t2g_3col}"
+  } else{
+      t2g_quant_path = t2g_path
+  }
+
```

Comment:
I would suggest moving these also to the start of the workflow, before the channels start.